src/client/pages/RetroCertsAuthPage/index.js

```diff
@@ -90,7 +90,9 @@ function RetroCertsAuthPage(props) {
     })
       .then((response) => response.json())
       .then((data) => {
-        recaptchaRef.current.reset();
+        if (recaptchaRef.current) {
+          recaptchaRef.current.reset();
```

Comment:
I'm getting errors in staging when trying to login because reset is being called on a null object.

Reply:
I get that too! I'm not sure why :/